rag/rag_app.py

```diff
@@ -75,7 +76,7 @@ def read_file(file):
 
 
 llm = ChatOpenAI(base_url=model_service, 
-                 api_key="EMPTY",
+                 api_key="EMPTY" if model_service_bearer is None else model_service_bearer,
```

Comment:
I know this was merged into the upstream so no point in making any changes here but just curious if we should have moved this conditional logic to a var above instead of the inline?

Reply:
For sure it adds the same functionality, tbh I consider both options equally verbose. We could open an extra PR to update this section in case we'd like to be this way.